src/app/client/src/app/modules/questionnaire/components/input-type-attachment/input-type-attachment.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { QuestionnaireService } from "../../questionnaire.service";
+
+@Component({
+  selector: "app-input-type-attachment",
+  templateUrl: "./input-type-attachment.component.html",
+  styleUrls: ["./input-type-attachment.component.scss"],
+})
+export class InputTypeAttachmentComponent implements OnInit {
+  @Input() data;
+
+  td = [
```

Comment:
what is the code for ?

Reply:
This was for testing. Removed hardcoded values.. In the next PR proper variable names will be reflected